checks/core/deployed_version.py

```diff
@@ -20,6 +20,6 @@ async def run(server: str, repo: str) -> CheckResult:
     latest_tag = releases["tag_name"]
 
     return (
-        deployed_version == latest_tag,
+        deployed_version.startswith(latest_tag),
```

Comment:
This check passes if version `1.1` is deployed and the latest version is `1.10`. How about
```suggestion
        deployed_version.partition("-")[0] == latest_tag,
```